includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -216,13 +218,22 @@ class DesignSystemGlobalFooterModel extends WikiaModel {
 		]
 	];
 
-	private $wikiId;
+	private $product;
+	private $id;
```

Comment:
Hmm, I'm wondering about renaming this to $productInstanceId - it's long, but describes the purpose of class var adequately